src/api.ts

```diff
@@ -111,6 +121,10 @@ export async function create(req: Request): Promise<Response> {
 		status: 'running',
 		source: req.source,
 		start_timestamp: new Date(),
+		is_final: !!req.is_final,
```

Comment:
Changed this to `is_final` for consistency

Reply:
@pipex Maybe since `req.is_final` is optional, maybe we could be doing either `req.is_final !== false` or `...(req.is_final != null && { is_final: req.is_final })` (so that the API is the one responsible to set the default value.
That's not an issue atm of course, since the CLI is using a shrinkwrap lock, so there is no chance we could have an old cli version running the new release-version package.